docs/advanced-usage/uninstallation.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Uninstallation
+description: 🗑 How to remove Spicetify.
+---
+
+## Windows
+
+### Command Prompt (Administrator)
```

Comment:
Does it really need to be admin command prompt? Also, we mostly recommend Powershell commands, so let's change to those.

Reply:
It does need to be in an admin shell, but i can change it to powershell now.